novuhq/website
# feat: updates for the Contributors page

**Describe what changes this pull request brings**
- illustration update in the `Hero` section
- adds section with `Issues`
- adds minor adjustments in styles
- adds a button that returns to the Contributors page

**Steps to test**
1. Open [the page](https://websitemain13551-contributorspageupdate.gtsb.io/contributors/)
2. Confirm that everything looks and works as expected
